cc/game_utils.cc

```diff
@@ -63,7 +63,7 @@ std::string FormatWinStatsTable(
 
 std::string GetOutputName(absl::Time now, size_t game_id) {
   return absl::StrCat(absl::ToUnixSeconds(now), "-", GetHostname(), "-",
-                      game_id);
+                      GetProcessId(), "-", game_id);
```

Comment:
I'm fairly sure I have regexes that depend on this. I should reconsider my life choices.

Reply:
sorry :(

Reply:
now you have two regexes!  Happy day.